0.data-download/README.md

```diff
@@ -9,20 +9,22 @@ https://depmap.org/portal/download/all/
 
 ## Files used in our multivariate gene dependency project
 
-### CRISPR_gene_dependency.csv
+### CRISPRGeneDependency.csv
```

Comment:
Updated file name based on new data set (22Q4)

Reply:
oh that's annoying that the file name changed! oh well, definitely a good thing for you to notice!!